client/src/components/MenuPost/style.scss

```diff
@@ -0,0 +1,57 @@
+@import '../../styles/variables';
+
+.burger-menu {
+  .dots {
+    position: absolute;
+    font-weight: bold;
+    font-size: 30px;
+    @media screen and (min-width: $break-md) {
```

Comment:
media queries should not be at the middle of the props

Reply:
But in all other styles it is like this and we decided to make the same styling everywhere, didn't we?